### DISCUSSION TITLE
Question: Where is the code that acutally determines the Filename & ModifiedDate to be written

### DISCUSSION THREAD

**Discussion by nealrobben** [DIGITALCRIMINALS/OnlyFans#894] [Q&A]
I am currently looking at the code to try and understand what's going on.
This is not so easy since I am a complete newby at Python.

What I am curious to see is the code that actually determines the filename to be written when downloading a photo/video from OF, and also how the ModifiedDate on the downloaded file is set.

I have looked at the code but not found it yet since the whole python syntax/structure is new to me.
Can anyone that is more familiar with the code point me in the right direction?

Thanks in advance!

**Comment by ondrovic** [comment]
I think you are looking for this
https://github.com/DIGITALCRIMINALS/UltimaScraperRenamer/blob/main/ultima_scraper_renamer/renamer.py

And then this
https://github.com/DIGITALCRIMINALS/UltimaScraperAPI/tree/main/ultima_scraper_api

Hope this helps
